src/app/client/src/environments/environment.desktop.staging.ts

```diff
@@ -2,5 +2,5 @@ export const environment = {
     env: 'desktop-staging',
```

Comment:
add Prod file as well and config in angular.json we will use that for prod build